lib/web/apiserver.go

```diff
@@ -2371,7 +2371,7 @@ func (h *Handler) getResetPasswordToken(ctx context.Context, tokenID string) (in
 //
 // POST /webapi/mfa/login/begin
 //
-// {"user": "alex", "pass": "abc123"}
+// {"user": "alex", "pass": "abcdef123456"}
```

Comment:
From what I tested, it is not possible to login with a previous password if it has less than 12 chars.

Reply:
Oof. @reedloden that would not be good as it would break existing users who are not in compliance with the new requirements. We will need to fix this if that's the case.

Reply:
Submitted #36579 to address. I had spun up a cloud instance to test this case prior to landing, but I had focused more on validating that we didn't have any client-side JS enforcing this (which is what I had seen in other products).